qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -563,45 +567,56 @@ def generate_hamiltonian(
     return convert_observable(h_of), nr_qubits
 
 
-def sd_excitations(n_electrons, n_orbitals, delta_sz=0):
-    r"""Generates single and double excitations from a Hartree-Fock (HF) reference state.
+def excitations(n_electrons, n_spin_orbitals, delta_sz=0):
+    r"""Generates single and double excitations from a Hartree-Fock reference state.
 
-    The singly- and doubly-excited configurations are generated by acting with the operators
-    :math:`\hat T_1` and :math:`\hat T_2` on the HF state:
+    Single and double excitations can be generated by acting with the excitation operators
+    :math:`\hat T_1` and :math:`\hat T_2` on the Hartree-Fock (HF) reference state:
 
     .. math:
-        && \vert \Phi_\mathrm{S} \rangle = \hat{T}_1 \vert \mathrm{HF} \rangle = \sum_{r \in
-        \mathrm{occ} \\ p \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_r \vert \mathrm{HF}
-        \rangle \\
-        && \vert \Phi_\mathrm{D} \rangle = \hat{T}_2 \vert \mathrm{HF} \rangle = \sum_{r>s \in
-        \mathrm{occ} \\ p>q \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_q^\dagger
-        \hat{c}_r \hat{c}_s \vert \mathrm{HF} \rangle
+
+        && \hat{T}_1 \vert \mathrm{HF} \rangle = \sum_{r \in \mathrm{occ} \\ p \in \mathrm{virt}}
+        \hat{c}_p^\dagger \hat{c}_r \vert \mathrm{HF} \rangle \\
+        && \hat{T}_2 \vert \mathrm{HF} \rangle = \sum_{r>s \in \mathrm{occ} \\ p>q \in
+        \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_q^\dagger \hat{c}_r \hat{c}_s
+        \vert \mathrm{HF} \rangle,
+
 
 	where the indices :math:`r, s` and :math:`p, q` run over the occupied (occ) and unoccupied,
-	referred to as virtual (virt), molecular orbitals and :math:`\hat c` and
+	referred to as virtual (virt), *spin-orbitals* and :math:`\hat c` and
 	:math:`\hat c^\dagger` are the electron annihilation and creation operators, respectively.
 
-    **Example**
+    |
 
-    >>> ph, pphh = sd_configs(2, 4, 0)
-    >>> print(ph)
-    [[0, 2], [1, 3]]
-    >>> print(pphh)
-    [[0, 1, 2, 3]]
+    .. figure:: ../../_static/qchem/sd_excitations.png
+        :align: center
+        :width: 70%
+
+    |
 
     Args:
-        n_electrons (int): number of active electrons
-        n_orbitals (int): number of active orbitals
-        delta_sz (int): optional argument to specify the spin-projection selection rule.
-            For single excitations ``sz[p] - sz[r] = delta_sz``.
-            For double excitations ``sz[p] + sz[p] - sz[r] - sz[s] = delta_sz``.
-            ``sz`` is the single-particle state spin quantum number and ``delta_sz``, in the
-            case of singles and doubles, can take the values :math:`0`, :math:`\pm 1`
-            and :math:`\pm 2`.
+        n_electrons (int): Number of electrons. If an active space is defined, 'n_electrons'
```

Comment:
Also noticing the capital letters here, i.e., "Number" vs "number".

As a side note, I like it, but I feel it's different to the current PL precedent

Reply:
I will be checking for this. I think in this case is OK based on the guidelines here: https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#punctuation-and-spelling

Let me know if I am missing something.